src/cli/serve/compile_config_stack.js

```diff
@@ -51,6 +51,19 @@ export function compileConfigStack({ configOverrides, devConfig, dev, serverless
     }
   }
 
+  if (serverlessMode === 'security') {
+    // Security specific tier configs
+    const serverlessSecurityTier = getSecurityTierFromCfg(configs);
+    if (serverlessSecurityTier) {
+      configs.push(resolveConfig(`serverless.${serverlessMode}.${serverlessSecurityTier}.yml`));
+      if (dev && devConfig !== false) {
+        configs.push(
+          resolveConfig(`serverless.${serverlessMode}.${serverlessSecurityTier}.dev.yml`)
+        );
+      }
+    }
+  }
```

Comment:
Could we test this logic in: https://github.com/elastic/kibana/blob/main/src/cli/serve/compile_config_stack.test.js

Reply:
Thanks @semd I'll update the tests